crates/cxx-qt-lib/src/core/mod.rs

```diff
@@ -142,4 +149,4 @@ mod ffi {
 /// ```
 ///
 /// See: <https://github.com/dtolnay/cxx/issues/1049>
-pub use ffi::c_void;
+pub use ffi::c_void;
```

Comment:
can you configure your editor to end with a new line so it is like before? (same on the other file)